test/integration/test_bamboo_relay_market_coordinated.py

```diff
@@ -285,20 +286,20 @@ def test_batch_market_buy(self):
         amount: float = 0.02
         current_price: float = self.market.get_price(symbol, False)
```

Comment:
We've changed `get_price()` in market to `Decimal` recently. So this kind of type hints would need to be changed in the test case as well.